### DISCUSSION TITLE
when save PageImage as PNG, Chinese comma looks strange

### DISCUSSION THREAD

**Discussion by bugm** [jsvine/pdfplumber#894] [Ask for help with specific PDFs]
Hello all, I am trying to extract the math formula from pdf page and save it as an image.
suppose I have gotten the (left, top, right, bottom) for the math formula part, I use the code to save it as image,
```
def save_image(page,(left, top, right, bottom)):
     page = page.crop((left, top, right, bottom))
     im = page.to_image(resolution=300)
     im.save("image.png",quality=95)
```
since I am dealing with some Chinese pdfs, which means there are Chinese comma in math formula, but the comma in the saved image looks a little strange.

This is the image saved

![test_1](https://github.com/jsvine/pdfplumber/assets/11450151/4c37ca92-4af4-4d9d-8f49-5e8e3097dcbe)

and this is the original formula in pdf
![result](https://github.com/jsvine/pdfplumber/assets/11450151/beaddf61-4bb5-4446-84fe-e921cd4ae6cf)

In the saved image, the comma looks like shifts right.

**Comment by jsvine** [comment]
Hi @bugm, and thanks for providing this example. Image conversion in `pdfplumber` uses [Wand](https://docs.wand-py.org/en/0.6.11/), which in turn is based on [ImageMagick](https://imagemagick.org/index.php), which in turn [uses Ghostscript](https://legacy.imagemagick.org/discourse-server/viewtopic.php?t=36461). So my guess is that this is not directly fixable by `pdfplumber`, unfortunately. 

But here's a test we can do: Try converting the PDF to PNG directly via ImageMagick: `convert file.pdf file.png`. Do you see the same problem, or no?
